webapp/src/App.tsx

```diff
@@ -205,7 +221,14 @@ const Chat = ({
             {appState === AppState.ErrorLoadingUserInfo && (
                 <Error text={'Oops, something went wrong. Please try signing out and signing back in.'} />
             )}
-            {appState === AppState.LoadingChats && <Loading text="Loading Chats..." />}
+            {appState === AppState.ErrorLoadingAuthInfo && (
+                <Error text={'Oops, unable to authentication info. Please try refreshing the page.'} />
```

Comment:
Maybe "Unable to load authentication info[...]" or "Unable to authenticate user[...]"?

Also, I would remove the "Oops" from all the strings here.

Reply:
ah yes, typo, it should be "Unable to **load** ...".